server/gql/_shared.graphql

```diff
@@ -0,0 +1,118 @@
+scalar Upload
+scalar Any
+scalar DateTime
+scalar URL
+scalar Lang
+scalar FileSize
+scalar TranslatedString
+scalar Cursor
+
+directive @goModel(
+  model: String
+  models: [String!]
+) on OBJECT | INPUT_OBJECT | SCALAR | ENUM | INTERFACE | UNION
+
+directive @goField(
+  forceResolver: Boolean
+  name: String
+) on INPUT_FIELD_DEFINITION | FIELD_DEFINITION
```

Comment:
```suggestion
```

Please deprecate these directive and use `gqlgen.yml` instead to enforce generating resolvers

Reply:
Done